HeterogeneousCore/SonicTriton/scripts/cmsTriton

```diff
@@ -346,9 +347,22 @@ edit_model(){
 	IFS='/' read -ra ADDR <<< "$MODELNAME"
 	CONFIG=$TMPDIR/$LOCALMODELREPO/${ADDR[-1]}/config.pbtxt
 	
+	MODELLOC=$(echo ""${ADDR[@]:0:${#ADDR[@]}-1} | sed "s/ /\//g")
+
 	PLATFORM=$(grep -m 1 "^platform:" "$CONFIG")
-	
-	cat <<EOF >> $CONFIG
+
+	if [[ $PLATFORM == *"ensemble"* ]]; then
+		SUBNAME=$(grep "model_name:" "$CONFIG")
+		for s in ${SUBNAME}; do
+			if ! [[ $s == *"model_name"* ]]; then
+				submodel=$(echo $s | tr -d '"')
+				ENSEMBLEMODELS+=($(echo ${MODELLOC}/$s | tr -d '"'))
+			fi
+		done
+	fi
+
+	if ! [[ $PLATFORM == *"ensemble"* ]]; then	
```

Comment:
should be elif

Reply:
I think this one should be an else.  I want a different outcome for ensemble or !ensemble, but checks for onnx/tensorflow are separate from that